src/handlers/motions/motionCreated/helpers.ts

```diff
@@ -14,22 +13,25 @@ import {
 
 export const getParsedActionFromMotion = async (
   motionId: string,
-  colonyClient: AnyColonyClient,
+  colonyAddress: string,
+  clients: NetworkClients[],
 ): Promise<TransactionDescription | undefined> => {
-  const votingClient = await colonyClient.getExtensionClient(
-    Extension.VotingReputation,
-  );
-
+  const votingClient = await getVotingClient(colonyAddress);
   const motion = await votingClient.getMotion(motionId);
 
-  try {
-    return colonyClient.interface.parseTransaction({
-      data: motion.action,
-    });
-  } catch {
-    verbose(`Unable to parse ${motion.action}`);
-    return undefined;
+  for (const client of clients) {
```

Comment:
We don't know which client is going to be needed to parse the transaction just from the event data... right?

On this assumption, I'm looping through the set of possible clients and returning the first time we don't get an error.

Reply:
Correct, and this is the right approach.

We have done this in the past, both for motions and both for normal events